lib/haml_lint/reporter/utils.rb

```diff
@@ -0,0 +1,99 @@
+module HamlLint
+  class Reporter
+    # Formatting helpers for printing the default report format.
+    module Utils
+      # Pluralizes a word based on a count.
+      #
+      # @param word [String] the word to pluralize
+      # @param count [Integer] the count of items
+      # @return [String]
+      def pluralize(word, count: 1)
+        if count.zero? || count > 1
+          "#{count} #{word}s"
+        else
+          "#{count} #{word}"
+        end
+      end
+
+      # Prints the lint with its location and severity.
+      #
+      # @param lint [HamlLint::Lint] the lint to print
+      # @return [void]
+      def print_lint(lint)
+        print_location(lint)
+        print_type(lint)
+        print_message(lint)
+      end
+
+      # Prints the location of a lint.
+      #
+      # @param lint [HamlLint::Lint] the lint to print
+      # @return [void]
+      def print_location(lint)
+        log.info lint.filename, false
+        log.log ':', false
+        log.bold lint.line, false
+      end
+
+      # Prints the severity of a lint.
+      #
+      # @param lint [HamlLint::Lint] the lint to print
+      # @return [void]
+      def print_type(lint)
+        if lint.error?
+          log.error ' [E] ', false
```

Comment:
Good call. I just pulled this directly from DefaultReporter without changes so didn't think about that.